pnpm-lock.yaml

```diff
@@ -9598,32 +9652,32 @@ packages:
       canvas:
         optional: true
     dependencies:
-      abab: 2.0.6
+      abab: 2.0.5
```

Comment:
saw a lot of version numbers are decreased, is it expected?

Reply:
fixed